src/OpenSpaceToolkit/Astrodynamics/Trajectory/Sequence.cpp

```diff
@@ -243,6 +243,15 @@ Sequence::Solution Sequence::solve(
     {
         for (const Segment& segment : segments_)
         {
+            if (segment.accessEventCondition()->getTarget().type_ == EventCondition::Target::Type::RelativeSegmentStart)
```

Comment:
```suggestion
            if (segment.accessEventCondition()->getTarget().getType() == EventCondition::Target::Type::RelativeSegmentStart)
```

nit: ditto here. Consider a getter/accessor for retrieving underscored variables